src/PrestaShopBundle/ApiPlatform/Resources/Hook.php

```diff
@@ -61,18 +61,21 @@
                     [
                         'name' => 'Authorization',
                         'in' => 'scopes',
-                        'description' => 'read:hook-status <br> write:hook-status ',
+                        'description' => 'hook-status_read <br> hook-status_write ',
                     ],
                 ],
             ],
             exceptionToStatus: [HookNotFoundException::class => 404],
             normalizationContext: [
                 'groups' => [
-                    'hook-status:read',
+                    'hook-status_read',
```

Comment:
```suggestion
                    'hook_status_read',
```

Reply:
For normalization groups though you can use a specific value since it's not a scope, however we should probably use a snake case convention as well and avoid mixing snake case and khebab case

Reply:
Also, I have one question regarding this normalization group, is it really needed? I believe when we discussed this with MAxime he explained to me that these groups were overkill because APIPLatform is able to return a partial DTO when some of its fields are null

The groups were added as an example for normalization but I actually think that if they're not needed we should remove them and keep the endpoints as clean and small and possible. Besides it would validate that we don't need normalization groups to handle partial APIs simply which is actually a more interesting feature than this advanced one based on groups